src/ticks/ticks.c

```diff
@@ -582,14 +583,34 @@ static const uint8_t mirror_table[] = {
     0x3, 0xB, 0x7, 0xF   /* 12-15 */
 };
 
+void exit_log(int code, char *fmt, ...)
+{
+    va_list  ap;
+    
+    char *program_name = (strrchr(__FILE__, '/') ? strrchr(__FILE__, '/') + 1 : __FILE__);
```

Comment:
This will report the program_name as `ticks.c`, on windows (if compiled with MSVC) it would report the full path to the file, for example: `c:\z88dk\src\ticks\ticks.c`.

Given that, it might be best just to hardcode "ticks" as program_name.

Reply:
Indeed, we'll go hardcore ;-)

I'll change and do an other pull request.

Reply:
Promblem with already applied changes in master repo should be solved.